infra/src/create-db.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { execSync } from 'child_process'
+import { createDb } from './lib/db'
+import { Pool } from 'pg'
+
+type ContainerConfig = {
+  containerName: string
+  masterUser: string
+  masterPassword: string
+  masterDatabase: string
+  host: string
+  sourcePort: number
+  targetPort: number
+}
+
+const createDefaultContainerConfig = (
+  overrides?: Partial<ContainerConfig>,
+): ContainerConfig => {
+  return {
+    containerName: 'create-db-test',
+    masterPassword: 'masterpassword',
+    masterUser: 'root',
+    masterDatabase: 'postgres',
+    host: 'localhost',
+    sourcePort: 5432,
+    targetPort: 5432,
+    ...overrides,
+  }
+}
+
+const startDockerContainer = (params: ContainerConfig) => {
```

Comment:
Do you know about [testcontainers](https://github.com/testcontainers/testcontainers-node). We are already using it in our repo if test suites require testing against PG instead of SQLite